### DISCUSSION TITLE
Using API Token in Script

### DISCUSSION THREAD

**Discussion by rs-follow** [codemagic-ci-cd/codemagic-docs#1417] [Q&A] [answered]
In Build A's post-publish, I want to do a curl request to trigger Build B. Is the API token available on the build machine, or will I need to pass that in as a custom env var?

**Comment by icarusdust** [comment, marked as answer]
Hi @rs-follow, your API token is account specific, so it is available. Though you are sill recommended to put that token in an env var anyways because you would not want to expose it.

**Reply by alexwastl** [reply]
Hi @icarusdust, is it also possible to re-generate the API token for the codemagic user account (i.e. accidentally pushed the api key to git repository)?

**Reply by icarusdust** [reply]
Hi @alexwastl, please share your user id. NOT THE API TOKEN but the personal account user id that can be found in the url when going to Teams >> Personal account.

**Reply by alexwastl** [reply]
@icarusdust the user id would be: 6461e68adad16a5e63957e30. Thanks a lot!

**Reply by icarusdust** [reply]
@alexwastl, updated successfully!

**Reply by alexwastl** [reply]
@icarusdust Perfect. Thanks a million for the support.
